src/components/newRepository.svelte

```diff
@@ -27,21 +26,15 @@
 				<Textfield label="Repository Name" type="email" />
 			</div>
 			<div class="mt-2 w-full">
-				<label for="textarea" class="block text-md text-gray-800"
-					>Description</label
-				>
+				<label for="textarea" class="block text-md text-gray-800">Description</label>
 				<textarea
 					rows="3"
 					class="w-full resize-none rounded-md focus:border-brown-600 focus:outline-brown-100 focus:ring-brown-700"
 				/>
 			</div>
-			<div class="flex px-16 pt-8 w-full">
-				<Button styles="text-gray-50 w-full mr-2" label="Create" />
-				<Button
-					onClick={() => toggleModal()}
-					styles="bg-gray-50 text-gray-800 w-2/3 ml-2"
-					label="Close"
-				/>
+			<div class="flex px-16 pt-8 w-full justify-center items-center space-x-5">
+				<ButtonSolid>Create</ButtonSolid>
+				<ButtonOutlined onClick={() => toggleModal()}>Close</ButtonOutlined>
```

Comment:
@guacamole we don't need the arrow func here, just `toggleModal` would do

Reply:
you got it!